### PR TITLE
[Feature #142465237] Add Read-Only API for exercise information

### PR DESCRIPTION
**Pull request by layodi**
#### What does this PR do? 
Add a new read-only API endpoint that includes all the information on an exercise

#### Description of Task to be completed? 
Have the following endpoint working:

``` GET api/v2/exerciseinformation/```

#### How should this be manually tested? 
Run ```python manage.py runserver``` in the terminal.
On the wger dashboard, open the ```REST API``` section under ```About this software``` on the navigation bar.
Click on the ```Browse the API``` button and search for the ```exerciseinformation``` link to view.

#### Any background context you want to provide? 
None

#### What are the relevant pivotal tracker stories? 
#142465237

#### Screenshots (if appropriate) #### 
screen shot 2017-04-23 at 11 21 36

Questions:
None

### CODE REVIEW

**Comment by amwaleh** [comment]
Nice work !!! add tests for the route and will be good to go

**Comment by layodi** [comment]
I am having a difficulty with writing the test for the route, the format I used keeps bringing errors but I will consult tomorrow to get unblocked.

File: wger/exercises/api/views.py [hunk 1]
```diff
@@ -206,3 +208,23 @@ class MuscleViewSet(viewsets.ReadOnlyModelViewSet):
...
+    filter_fields = ('category',
+                     'creation_date',
+                     'description',
+                     'language',
+                     'muscles',
+                     'muscles_secondary',
+                     'name',
+                     'equipment')
+
+    def exercise_information(request):
```
Comment (brayovsky) [inline wger/exercises/api/views.py]: You could also consider using methods such as `.list()`, `.create()` etc to handle different http methods. Is this method even used?

Comment (layodi) [reply wger/exercises/api/views.py]: I looked through it and I realized the method was redundant and I have excluded it.

**Review by amwaleh — APPROVED** [review]
great work

**Pull request merged**